docker/istio-dev/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM docker:stable as docker
+
+FROM ubuntu:xenial
+ARG goversion=1.12.5
+ARG user
+ARG group
+ARG uid=1000
+ARG gid=1000
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Install development packages.
```

Comment:
Can you ensure that it has everything from here too?
https://github.com/istio/proxy/blob/master/.circleci/Dockerfile

Reply:
@mandarjog so you want bazel and jdk in the build-tools image?  I thought we had punted on the proxy repo for the moment - to tackle at a later time.

Reply:
@sdake This is not build-tools, this is istio-dev.

I think the proxy repo needs to be on a completely different plan than the rest of the project. So I'd rather not pull stuff into build-tools for the proxy repo until such a point that we have a good plan.